lib/asciidoctor/syntax_highlighter.rb

```diff
@@ -0,0 +1,80 @@
+module Asciidoctor
+  module SyntaxHighlighter
+    PROVIDED = RUBY_ENGINE_OPAL ? {} : {
+      'coderay' => 'syntax_highlighter/coderay',
+      'highlight.js' => 'syntax_highlighter/highlightjs',
+      'highlightjs' => 'syntax_highlighter/highlightjs',
```

Comment:
I don't think we should do it in this pull request but we should deprecate `highlightjs` ?

Reply:
Yeah, let's address that separately. It's tricky. I wish the library had an unambiguous name :/

Reply:
Since we load it eagerly, we actually defer this problem a bit. I'd still like to align on a single name, though, because it's a pain to have to deal with the exception.